metadata_dictionary/experiment_dict.tsv

```diff
@@ -0,0 +1,28 @@
+Field	Attribute	Description	Permissible Values	Note
+type	Required	table type	sequencing_experiment	
+submitter_sequencing_experiment_id	Required	Unique identifier of the sequencing experiment, assigned by the data provider.	String values that meet the regular expression ^[a-zA-Z0-9]{1}[a-zA-Z0-9\\-_\\.:']{0,98}[a-zA-Z0-9]{1}$	
+program_id	Required	ARGO Program ID, the unique identifier of your program. If you have logged into the ARGO Data Platform, this is the Program ID that you see in the Program Services area.		Must be the same as what is in sample_registration table
+submitter_donor_id	Required	Unique identifier of the donor, assigned by the data provider.	Values must meet the regular expression  ^[A-Za-z0-9\-\._]{1,64}	Must be the same as what is in sample_registration table
+gender	Required	Description of the donor self-reported gender. Gender is described as the assemblage of properties that distinguish people on the basis of their societal roles.	Female Male Other	Must be the same as what is in sample_registration table
+submitter_specimen_id	Required	Unique identifier of the specimen, assigned by the data provider.	Values must meet the regular expression ^[A-Za-z0-9\-\._]{1,64}	Must be the same as what is in sample_registration table
+specimen_tissue_source	Required	Tissue source of the biospecimen.	Blood derived - bone marrow, Blood derived - peripheral blood, Blood derived, Bone marrow, Bone. Buccal cell,Buffy coat,Cerebellum,Cerebrospinal fluid,Endometrium,Esophagus,Intestine,Lymph node,Mononuclear cells from bone marrow,Other,Plasma,Pleural effusion,Saliva,Serum,Skin,Solid tissue,Spleen,Sputum,Stomach,Tonsil,Urine	Must be the same as what is in sample_registration table
+tumour_normal_designation	Required	Description of specimens tumour/normal status for data processing.	Normal Tumour	Must be the same as what is in sample_registration table
+specimen_type	Required	Description of the kind of specimen that was collected with respect to tumour/normal tissue origin.	 Cell line - derived from normal  Cell line - derived from tumour  Cell line - derived from xenograft tumour  Metastatic tumour - additional metastatic  Metastatic tumour - metastasis local to lymph node  Metastatic tumour - metastasis to distant location  Metastatic tumour  Normal - tissue adjacent to primary tumour  Normal  Primary tumour - additional new primary  Primary tumour - adjacent to normal  Primary tumour  Recurrent tumour  Xenograft - derived from primary tumour  Xenograft - derived from tumour cell line	Must be the same as what is in sample_registration table
+submitter_sample_id	Required	Unique identifier of the sample, assigned by the data provider. If submitted along with BAM molecular data, must also be present in header SM, and values must meet the regular expression  ^[A-Za-z0-9\-\._]{1,64}	Must be the same as what is in sample_registration table

```

Comment:
For this row looks like column `permissible values` and `description` was combined, please separate